programs/ssl/ssl_client2.c

```diff
@@ -1949,7 +2007,22 @@ int main( int argc, char *argv[] )
         }
 #endif
 
-    mbedtls_ssl_conf_rng( &conf, mbedtls_ctr_drbg_random, &ctr_drbg );
+    if (opt.reproducible)
+    {
+        srand( 1 );
+        mbedtls_ssl_conf_rng( &conf, dummy_random, &ctr_drbg );
```

Comment:
Please keep using `mbedtls_ctr_drbg_random`, just seed it with dummy entropy.

Reply:
Ok if you are sure